configurations/validateConfig.go

```diff
@@ -38,27 +37,21 @@ func validateBaseImage(baseImage string) error {
 	if !found {
 		return errors.New("environment variable for username not set")
 	}
-	password, found := os.LookupEnv(environment.DockerAuthPassword)
-	if !found {
-		return errors.New("environment variable for password not set")
-	}
-	authConfig := types.AuthConfig{
-		Username: username,
-		Password: password,
-	}
-	encodedJSON, err := json.Marshal(authConfig)
-	if err != nil {
-		return errors.Wrap(err, "error in encoding authConfig")
-	}
-
-	authStr := base64.URLEncoding.EncodeToString(encodedJSON)
-
-	_, err = dockerClient.ImageSearch(backgroundContext, baseImage, types.ImageSearchOptions{
-		RegistryAuth: authStr,
-		Limit:        10})
 
+	resp, err := dockerClient.ImageSearch(backgroundContext, username, types.ImageSearchOptions{
+		Limit: 10})
```

Comment:
what is the reason for choosing 10 as the limit?

Reply:
I have changed it to 25. The reason for keeping it 25 is the default limit while searching image through command line is 25. But through the docker client function it was taking limit as 0 as default. So I set it to 25.